cmd/vgmanager/vgmanager.go

```diff
@@ -109,8 +115,10 @@ func run(cmd *cobra.Command, _ []string, opts *Options) error {
 		LeaderElection:         false,
 		Cache: cache.Options{
 			DefaultNamespaces: map[string]cache.Config{
-				os.Getenv("POD_NAMESPACE"): {},
+				operatorNamespace: {},
 			},
+			ByObject: map[client.Object]cache.ByObject{
```

Comment:
I dont thing its necessary to include this in order to be cached. You have a watches and that should already include it for the namespace